src/browser/karma.conf.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(config) {
       // suppress annoying 404 warnings for resources, images, etc.
       { pattern: 'dist/dev/assets/**/*', watched: false, included: false, served: true },
 
-      'test-main.js'
+      'src/browser/test-main.js'
```

Comment:
Thanks for the info! I updated the build task accordingly with the commit https://github.com/mgechev/angular2-seed/pull/926/commits/0129daca74c42c9f39da348c25e08069ac579d5e